sparkse.py

```diff
@@ -0,0 +1,157 @@
+#################################################################################
+# Author: Eric Sparks
+# Username: sparkse
+#
+# Assignment: HW03 Gitty
+# Purpose:
+# Google Doc Link:
+# 1
+#################################################################################
+# Acknowledgements:
+#  https://stackoverflow.com/questions/20320921/python-turtle-pen-colour
+#  Changed color to 255 type
+#################################################################################
+
+
+import turtle
+import math
+import random
+
+
```

Comment:
Pay attention to gray squigglies, they indicate style guidelines. This one indicates there should be exactly two blank lines before function definitions.